src/init.cpp

```diff
@@ -1449,6 +1449,7 @@ bool AppInitMain()
                 // Note that it also sets fReindex based on the disk flag!
                 // From here on out fReindex and fReset mean something different!
                 if (!LoadBlockIndex(chainparams)) {
+                    if (fRequestShutdown) break;
```

Comment:
Also use `ShutdownRequested()`? I realize that`fRequestShutdown` is used in `src/init.cpp` (the function is also used) but I wonder if using `ShutdownRequested()` should be used instead.

Reply:
Ping.